kivy/core/text/markup.py

```diff
@@ -752,7 +752,14 @@ def p_restricted(line, uw, c):
         s = self.get_extents(last_text)
         if len(last_text):
             line1.append(LayoutWord(last_word.options, s[0], s[1], last_text))
-        elps.options = last_word.options
```

Comment:
Hmm, I cannot remember why I overwrite the ellipse options with the last word's options. As far as I remembered the ellipse should be the same opts as the main text, but this doesn't seem to comply with that. Unless the last word's opts is the same as the main text, but then I'm not sure why I overwrote it here.